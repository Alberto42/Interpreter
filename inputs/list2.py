l = [1,2,3]
l2 = [l,l]

print(l2)