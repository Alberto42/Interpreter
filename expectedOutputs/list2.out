Ident "l2" = [[1,2,3],[1,2,3]]
Ident "l" = [1,2,3]

[[1,2,3],[1,2,3]]

